benches/parse_preprocessed_bench.rs

```diff
@@ -0,0 +1,23 @@
+extern crate lang_c;
+
+use lang_c::driver;
+use std::time::Instant;
+
+fn main() {
+    let mut _config = driver::Config::with_clang();
+    #[cfg(target_os = "macos")]
+    {
+        _config.cpp_options.push("-fno-blocks".into());
```

Comment:
`cpp_options` are not used when calling `parse_preprocessed`. I think this block can be removed.